src/pages/services/aem-universal-editor/api/properties-rails/index.md

```diff
@@ -0,0 +1,162 @@
+
+![Properties Rail](properties-rail-initial.png)
+
+## Example of adding custom panels
+
+This code snippets demonstrate how to create a custom panel using [UIX SDK](https://github.com/adobe/uix-sdk) library and add it to the properties rail of the editor, enabling users to access and interact with the custom functionality seamlessly.
+
+```js
+// App.js
+
+import { HashRouter as Router, Routes, Route } from "react-router-dom"
+import ExtensionRegistration from "./ExtensionRegistration"
+import RailContent from "./RailContent";
+
+// ...
+
+function App() {
+    return (
+        <Router>
+            <ErrorBoundary onError={onError} FallbackComponent={fallbackComponent}>
+                <Routes>
+                    <Route index element={<ExtensionRegistration />} />
+                    <Route
+                        exact
+                        path="index.html"
+                        element={<ExtensionRegistration />}
+                    />
+                    <Route
+                        exact path="rail/:railId"
+                        element={<RailContent />}
+                    />
+                </Routes>
+            </ErrorBoundary>
+        </Router>
+    )
+    // ...
+}
+```
+
+```js
+// ExtensionRegistration.js
+
+import { register } from "@adobe/uix-guest";
+
+// ...
+
+function ExtensionRegistration() {
+    useEffect(() => {
+        const init = async () => {
+            const registrationConfig = {
+                id: extensionId,
+                methods: {
+                    rightPanel: {
+                        addRails() {
+                            return [
+                                {
+                                    id: "my.company.panel_1",
+                                    header: "Last Changes",
+                                    url: '/#/rail/1',
+                                    icon: 'Export',
+                                },
+                                {
+                                    id: "my.company.panel_2",
+                                    header: "Workflow",
+                                    url: '/#/rail/2',
+                                    hotkey: "w",
+                                    icon: 'Import',
+                                },
+                            ];
+                        },
+                    },
+                },
+            };
+            const guestConnection = await register(registrationConfig);
+        }
+        init().catch(console.error)
+    }, []);
+    return <Text>IFrame for integration with Host...</Text>
+}
+```
+
+```js
+// RailContent.js
+
+import { attach } from "@adobe/uix-guest";
+
+// ...
+
+export default () => {
+    const { railId } = useParams();
+    if (!railId) {
+        console.error('Rail id parameter is missed');
+        return;
+    }
+
+    // If you need to interact with an AEM instance
+    // const connection = await attach({ id: extensionId });
```

Comment:
Not having this thing will throw the error in the console

Reply:
fixed